app/routes/docs/circulars/archive.mdx

```diff
@@ -24,3 +24,12 @@ The [GCN Viewer](https://heasarc.gsfc.nasa.gov/wsgi-scripts/tach/gcn_v2/tach.wsg
 ## Migration of Archive from GCN Classic
 
 The GCN Circulars archive was migrated from the [GCN Classic Circulars archive page](https://gcn.gsfc.nasa.gov/gcn3_archive.html) on April 17, 2023. The [new GCN Circulars archive on this site](/circulars) is the authoritative record of all past Circulars and is updated in real time with new Circulars. The old GCN Classic Circulars archive page is preserved, but is no longer updated.
+
+During the migration, we made some minor adjustments to some Circulars to adapt them to the structured format of the new database:
+
+- There were two instances where the old archive added a letter suffix to a Circular ID to distinguish multiple Circulars that were initially assigned the same ID. In the new archive, we have replaced these with half-integer IDs. [GCN 18448a](https://gcn.gsfc.nasa.gov/gcn3/18448a.gcn3) became [GCN 18448.5](/circulars/18448.5) and [GCN 18453a](https://gcn.gsfc.nasa.gov/gcn3/18453a.gcn3) becamse [GCN 18453.5](/circulars/18448.5).
+- Some Circulars listed both a "date" and a "revised submission date". Where applicable, the date presented in the new archive is the revised submission date.
+- The very earliest Circulars (before [GCN 31](/circulars/31)) did not record a subject, submission date, or submitter email address. Where these fields were missing, we set the subject to `Untitled`, the date to January 1, 1970 (the start of the [UNIX epoch](https://en.wikipedia.org/wiki/Unix_time)), and the submitter email address to `circulars@gcn.nasa.gov`.
+- Some Circulars submitted by Scott Barthelmy did not display an email address. In these cases, we have rendered the submitter as `Scott Barthelmy at NASA/GSFC <scott@lheamail.gsfc.nasa.gov>`.
+- Some Circulars have editorial remarks indicatd by the text `[GCN OP NOTE ...]`. In most cases, this text was at the bottom of the Circular below the body text, but in a few cases it was at the top, before the headers. We have moved all `[GCN OP NOTE ...]` remarks to the bottom.
```

Comment:
```suggestion
- Some Circulars have editorial remarks indicated by the text `[GCN OP NOTE ...]`. In most cases, this text was at the bottom of the Circular below the body text, but in a few cases it was at the top, before the headers. We have moved all `[GCN OP NOTE ...]` remarks to the bottom.
```

Reply:
There were some other formats of these notes (e.g. "Editor's Note").  I've added some over the last year, and used a different syntax, and it looks like Scott had also varied it over time.